publications/ohbm_2024/brainhack_project.md

```diff
@@ -0,0 +1,100 @@
+
+Recent work in Nilearn has been centered on developing a new API to allow users to seamlessly work with surface data in a manner similar to volumetric data, enhancing support for the General Linear Model (GLM), enhancing the BIDS interface, and improving and updating the infrastructure and codebase.
+
+To this end, we aim to get contributors to get familiar with the package, improve and provide feedback on the new surface data functionality as well as the package overall. Contributors are encouraged to work on issues in the project board by opening pull requests. New contributors should look for the "Good First Issue" label on the project board to get started.
+```
+
+# Link to the project
+(Please provide **one URL** for the project, e.g. GitHub repository. See [the guide](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#link-to-project) for other suggestions.)
+```txt
+https://github.com/nilearn/nilearn
+```
+
+# Image/Logo for the OHBM brainhack website
+(Please provide a link only. Please provide the RAW content of the file if you use an image from a GitHub repo. We kindly request that the image be provided in a square format, ensuring equal width and height for a polished visual appearance.)
+```txt
+https://drive.google.com/file/d/1vgv_osZHtleHkGLztaSN-1SicwUhNBMQ/view?usp=sharing
+```
+
+# Project lead
+(Your name and GitHub login AND Discord login (we will be using Discord during the hackathon, please be sure to register [here](https://discord.gg/yK8jakPfnG)). Add multiple leaders if you can, if leaders cover different hubs, please add this next to the name.)
+```txt
+Himanshu Aggarwal, Github: @man-shu, Discord: man_shooo
+```
+
+# Main Hub
+(The main Hackathon Hub of your project, see more [here](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#hubs))
+Options
+- Seoul
+- Hybrid (Asia / Pacific)
+- Hybrid (Europe / Middle East / Africa)
+- Hybrid (Americas)
+```txt
+Seoul
+```
+
+# Link to the Project pitch
+(not required, but highly recommended) If your project involves hybrid/online participation, create a brief project pitch (2-5 minutes) for the Brainhack audience. See [more](https://github.com/ohbm/hackathon2024/blob/main/.github/ISSUE_TEMPLATE/handbooks/projects.md#pitch-video) for additional guidance. Upload the pitch video to YouTube and share the link with us.
+```txt
+```
+
+# Other hubs covered by the leaders
+(Please choose other hubs that are covered by your leaders (be sure to have at least one leader in the specific timezones, and make a note next to their names))
+- Seoul
+- Hybrid (Asia / Pacific)
+- Hybrid (Europe / Middle East / Africa)
```

Comment:
I probably can't dedicate a full day to the brainhack. So I'ml not sure it is reasonable to advertise it.

Reply:
Ok! That's what I wanted to know